### PR TITLE
Updated README.md.

### PR DESCRIPTION
**Pull request by cknitt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [![NPM](https://nodei.co/npm/bs-react-intl.png?compact=true)](https://nodei.co/npm/bs-react-intl/)
 
-[BuckleScript](https://bucklescript.github.io) bindings to [react-intl](https://github.com/yahoo/react-intl).
+[BuckleScript](https://bucklescript.github.io) bindings to [react-intl].
```
Comment (alex35mil) [inline README.md:5]: I think you accidentally removed url.

File: README.md [hunk 2]
```diff
@@ -24,29 +24,9 @@ yarn start
...
-- [x] DefinedMessage
-- [ ] DefinedMessage (children-as-function)
-- [ ] FormattedHTMLMessage
-- [ ] FormattedHTMLMessage (children-as-function)
+bs-react-intl should cover all of the [react-intl] 3.0.0 API. If you find anything missing, please file an issue.
 
 ## Usage
 See [`examples`](./examples) folder.
+
+[react-intl]: https://github.com/formatjs/react-intl
```
Comment (alex35mil) [inline README.md:32]: Or pasted it in the wrong place :)

Comment (cknitt) [reply README.md:32]: No, should be fine. It is defined at the bottom and referenced twice above.

Comment (alex35mil) [reply README.md:32]: Ah gotcha gotcha

**Pull request merged**